sqlglot/expressions.py

```diff
@@ -990,7 +990,7 @@ class Uncache(Expression):
     arg_types = {"this": True, "exists": False}
 
 
-class TableChange(Expression):
+class DDL(Expression):
```

Comment:
I didn't have `UPDATE` inherit from this because it doesn't have a `Subqueryable` expression like `CREATE` and `INSERT`, so it'd need more effort to qualify / optimize its `SET .. FROM ..` syntax.